__tests__/app.test.js

```diff
@@ -265,6 +266,35 @@ describe('/api/articles/:article_id/comments', () => {
             expect(comments).toBeSortedBy('created_at',{descending:true})
         })
     })
+
+    test('GET 200: responds with the total count of comments of given article_id', () => {
+        return request(app)
+        .get('/api/articles/9(comment_count)')
```

Comment:
So this endpoint wants to make sure you can go back and add in the comment count to the article that comes back from requesting /api/articles/:article_id. It is not convention to have an endpoint in the format of /api/articles/:article_id(comment_count). So for this task we dont need a new endpoint, we just need to create a new test to check that the comment_count property exists without changing any existing tests, as well as updating the original controllers and models in the /api/articles/:article_id endpoint. Hope that makes sense but give me a shout if it doesnt!